autoload/lsp/inlayhints.vim

```diff
@@ -157,6 +157,14 @@ export def BufferInit(lspserver: dict<any>, bnr: number)
 		group: 'LspInlayHints',
 		cmd: $'LspInlayHintsUpdateStop({bnr})'})
 
+  # Inlay hints maybe a bit delayed if it was a sync init lsp server.
+  if lspserver.syncInit
+    acmds->add({bufnr: bnr,
+		  event: ['User'],
+		  group: 'LspAttached',
+		  cmd: $'LspInlayHintsChanged({bnr})'})
```

Comment:
The LspAttached autocmd handler needs to be executed only once for this buffer.  So can you add `once: v:true` to the autocmd Dict?

Reply:
1, i donot know why, but once added `once: true`, that au may not be kicked-off.
2, normally bufinit would/should be done only once when addFile()?? unless manually `doau` it again.
3, `exists('#User#LspAttached')` seems should be `exists('#LspAttached#User')` but actually worked, weird.

Reply:
If `once` is not used, then the autocmd will be present forever (even though it will not be invoked).
If you run the `:autocmd` command, you will see too many of these.

Are you saying that if you set `once` to `true`, then the autocmd is not invoked?

Reply:
> If `once` is not used, then the autocmd will be present forever (even though it will not be invoked).
> If you run the `:autocmd` command, you will see too many of these

yes, since it's on buf, if that's a concern, then maybe a bit annoyed, but:


> Are you saying that if you set `once` to `true`, then the autocmd is not invoked?

yes, au was not invoked at all, no even once (nothing).